sig-scheduling/annual-report-2022.md

```diff
@@ -39,23 +39,23 @@
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Diversity 
+   - Number of contributors
+   - Meetings attendance 
```

Comment:
I think this counts as community health: comprehensive coverage at integration level and zero test flakiness.

wdty, @liggitt?

Reply:
I think "community health" is more oriented around healthy trajectories for people participating

Reply:
I see.

In that case, I would add: number of reviewers, which has stalled since last year :(